src/js/player.js

```diff
@@ -33,23 +55,23 @@ vjs.Player = vjs.Component.extend({
     this.tag = tag; // Store the original tag used to set options
 
     // Make sure tag ID exists
-    tag.id = tag.id || 'vjs_video_' + vjs.guid++;
+    tag.id = tag.id || 'vjs_video_' + Lib.guid++;
 
     // Store the tag attributes used to restore html5 element
-    this.tagAttributes = tag && vjs.getElementAttributes(tag);
+    this.tagAttributes = tag && Lib.getElementAttributes(tag);
 
     // Set Options
     // The options argument overrides options set in the video tag
     // which overrides globally set options.
     // This latter part coincides with the load order
     // (tag must exist before Player)
-    options = vjs.obj.merge(this.getTagSettings(tag), options);
+    options = Lib.obj.merge(this.getTagSettings(tag), options);
 
     // Update Current Language
-    this.language_ = options['language'] || vjs.options['language'];
+    this.language_ = options['language'] || options['language'];
 
     // Update Supported Languages
-    this.languages_ = options['languages'] || vjs.options['languages'];
+    this.languages_ = options['languages'] || options['languages'];
```

Comment:
oops, we have a shadowing issue here. The global and local options are called the same. We should change the global options to be called `opts` or something like that instead.